magefiles/dagger.go

```diff
@@ -26,7 +25,7 @@ const (
 	// https://hub.docker.com/r/flyio/flyctl/tags
 	flyctlVersion = "0.1.65"
 
-	appName          = "dagger-registry-2023-01-23"
+	appName          = "dagger-registry-2023-07-28"
```

Comment:
this will likely need a few other changes in our vector config, we can go through them together when we are ready to merge.